src/settings.cpp

```diff
@@ -41,6 +41,9 @@ namespace openmc {
 
 namespace settings {
 
+// DiscreteIndex class for the alias sampling
+
+DiscreteIndex external_sources_alias_sampler;
```

Comment:
Perhaps another misunderstanding - this line should go in a different file: `source.cpp` at line 46, not in this file